gcp_variant_transforms/libs/hashing_util.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates hashing code for string."""
+
+import sys
+
+import farmhash
+
+
+def generate_unsigned_hash_code(string, max_hash_value=sys.maxint):
```

Comment:
Could we modify this function to receive a list of strings and then serialize them as explained [here](https://jameshfisher.com/2018/01/09/how-to-hash-multiple-values/)?